committee-steering/governance/wg-governance.md

```diff
@@ -62,8 +60,7 @@ requirements for that are:
 - chair information
 - meeting information
 - contact methods
-- any sig stakeholders
-- any subproject stakeholders
+- any [sig or subproject](sig-governance.md#project-management) stakeholders
```

Comment:
dare I ask why sig or subproject? asking from a perspective of wanting to update `sigs.yaml` to have working groups specify which entities they interface with

I had assumed it was just going to be a `sigs:` field, now I'm guessing it would look like `sigs:` and `subprojects:` fields and wondering why the additional complexity

Reply:
(ref https://github.com/kubernetes/community/issues/2176)

Reply:
+1 to this question

Reply:
PTAL.  I removed the line about projects.